skyportal/tests/fixtures.py

```diff
@@ -228,14 +226,18 @@ class Meta(BaseMeta):
     filters = ['ztfg', 'ztfr', 'ztfi']
 
     @staticmethod
-    def teardown(instrument):
-        if is_already_deleted(instrument, Instrument):
-            return
-
-        telescope = instrument.telescope.id
-        DBSession().delete(instrument)
-        DBSession().commit()
-        TelescopeFactory.teardown(telescope)
+    def teardown(instrument_id):
+        instrument_ = (
```

Comment:
Just curious, why the underscore?

Reply:
Just imitating some other handlers that had this ;)